src/libs/Navigation/AppNavigator/ReportScreenIDSetter.ts

```diff
@@ -44,6 +46,12 @@ function ReportScreenIDSetter({route, reports, policies, navigation, isFirstTime
     useEffect(() => {
         // Don't update if there is a reportID in the params already
         if (route?.params?.reportID) {
+            const reportActionID = route?.params?.reportActionID;
+            const regexValidReportActionID = new RegExp(/^\d*$/);
+            if (reportActionID && !regexValidReportActionID.test(reportActionID)) {
+                Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(route?.params?.reportID));
+                return;
+            }
```

Comment:
```suggestion
            const reportActionID = route?.params?.reportActionID;
            const regexValidReportActionID = new RegExp(/^\d*$/);
            if (reportActionID && !regexValidReportActionID.test(reportActionID)) {
                navigation.setParams({reportActionID: ''});
            }
```

Sorry, but this works better.

Reply:
Can you change to this?